cadl-extension/src/emitter.ts

```diff
@@ -7,7 +13,24 @@ import { CodeModelBuilder } from "./code-model-builder.js";
 import { dirname } from "path";
 import { fileURLToPath } from "url";
 
-export async function $onEmit(program: Program) {
+export interface EmitterOptions {}
```

Comment:
Same here, we want exact type of possible options (like the code in tutorial). Not a blank `interface {}`.